google/ggrc-core
# GGRC-4220 Fix reindex Snapshots with Option without "title"

# Issue description
Reindex doesn't process Option without "title" in content for Snapshots

# Steps to test the changes
1. Create a Snapshot that references an old Control Revision*.
2. Run `reindex_snapshots` job.

* The "old Control Revision" should contain a predefined dropdown field implemented through Option, like "Kind" field on Controls, and revision.content["kind"] should not contain "title" field (only valid for revisions created before

# Solution description
Update reindex action and populate records with "title" from Options for full text search
# Sanity checklist




# PR Review checklist
